Dockerfile.fips

```diff
@@ -0,0 +1,52 @@
+# This version of Go is a Go+BoringCrypto release for FIPS 140-2 compliance
```

Comment:
This file may need to go to `build` folder once https://github.com/codenotary/immudb/pull/1366 gets merged